eradiate/scenes/measure.py

```diff
@@ -417,6 +436,17 @@ class PerspectiveCameraMeasure(Measure):
         units_compatible=cdu.generator("length"),
     )
 
+    origin = attrib_quantity(
+        default=ureg.Quantity([0, 0, 0], ureg.m),
+        validator=validator_has_len(3),
+        units_compatible=cdu.generator("length"),
+    )
+
+    up = attr.ib(
+        default=[0, 0, 1],
+        validator=validator_has_len(3)
+    )
+
     zenith = attrib_quantity(
```

Comment:
I'd like to get rid of zenith and azimuth parameters completely, the purpose of this interface change is to remove them in favour of something more flexible.